go/chat/commands/dm.go

```diff
@@ -0,0 +1,39 @@
+package commands
+
+import (
+	"context"
+	"strings"
+
+	"github.com/keybase/client/go/chat/globals"
+	"github.com/keybase/client/go/protocol/chat1"
+	"github.com/keybase/client/go/protocol/gregor1"
+)
+
+type DM struct {
+	*baseCommand
+}
+
+func NewDM(g *globals.Context) *DM {
+	return &DM{
+		baseCommand: newBaseCommand(g, "msg", "<conversation> <message>",
+			"Send a message in the specified conversation", "dm"),
+	}
+}
+
+func (d *DM) Execute(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID,
+	tlfName, text string) (err error) {
```

Comment:
why not pass `tokens []string` instead of `text`, cuts the boilerplate for each command to validating the tokens directly

Reply:
if not at least wrap the text splitting to override `commandAndMessage` to keep the `Execute` more streamlined

Reply:
How important is this to you? I feel like calling `string.Split` in here isn't so bad.

Reply:
just seems messy and wrapping in `commandAndMessage` (maybe called `tokenize`) or passing `tokens` would make it cleaner